WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -273,12 +272,16 @@ def run_summary(_, instrument_name=None, run_number=None, run_version=0):
             reduction_location = reduction_location.replace('\\', '/')
 
         rb_number = Experiment.objects.get(id=run.experiment_id).reference_number
+        has_variables = bool(
```

Comment:
Could you add a comment explaining why you check for both InstrumentVariablesUtils().get_default_variables(run.instrument.name) and run.run_variables.all() ?